analysis/boostedHbbProcessor.py

```diff
@@ -398,10 +403,14 @@ def postprocess(self, accumulator):
         # set everything to 1/fb scale
         lumi = 1000  # [1/pb]
 
+        normlist = self._corrections['sumw_external']
```

Comment:
Can you wrap this with a check that `sumw_external` is present, otherwise fall-through to the traditional version?